apps/omg/lib/omg/fees.ex

```diff
@@ -20,19 +20,21 @@ defmodule OMG.Fees do
   alias OMG.MergeTransactionValidator
   alias OMG.State.Transaction
   alias OMG.Utxo
+  alias OMG.WireFormatTypes
 
   require Utxo
 
   use OMG.Utils.LoggerExt
 
   @type fee_t() :: %{Transaction.Payment.currency() => fee_spec_t()}
+  @type full_fee_t() :: %{non_neg_integer() => fee_t()}
```

Comment:
Can you add a `@typedoc` at least for this one if not all? The `non_neg_integer()` key makes me wonder.

Reply:
Or add a new type that describes `non_neg_integer()`